steps.md

```diff
@@ -0,0 +1,272 @@
+# One Developer's View
+
+These are the steps you *might* take to complete this project. There's no right way to make an Express app -- nor any other app.
+
+## Get something on Github
+
+1. Create a new empty Github repository called "Project3". You can change the name later.
+- Create the skeleteon of your Node app:
+  ```bash
+  $ git clone git@github.com:myusername/project3.git
+  $ cd project3
+  $ npm init # Then hit 'return' a bunch of times
+  $ npm install --save body-parser express hbs method-override mongoose
+
+  $ mkdir db
+  $ touch db/schema.js
+  $ touch db/seeds.js
+
+  $ mkdir models
+  $ touch models/.keep
+
+  $ mkdir views
+  $ touch views/layout.hbs
+  $ touch views/index.hbs
+
+  $ mkdir controllers
+  $ touch controllers/.keep
+
+  $ mkdir public
+  $ mkdir public/js
+  $ touch public/js/script.js
+  $ touch public/styles.css
+
+  $ touch app.js
```

Comment:
After discussion: I'm going to leave it this way because someone's going to use `app.js`, and then they'll have Nodemon saying "I can't find `index.js`!" This way, they'll have an idea of how to deal with that.